doc/tutorials/ostro-oic-tutorial/step1/tutorial.md

```diff
@@ -0,0 +1,266 @@
+to subflow input port `IN_POWER`, this packet is sent to port IN of node
+`power`. Conversely, each time a packet is sent to node `power` OUT port,
+it is sent to subflow port `OUT_POWER`.
+Before finishing our server, let's print, for debug purposes, some
+information about our running server. It is just a matter of sending
+a constant string to `console` node type:
+
+```sh
+_(constant/string:value="http-light-server running on port 8002, paths: /state and /power") OUT -> IN _(console:prefix="note: ")
+```
+
+First node is a `constant/string` node, it's behaviour is just to send to
+its `OUT` port a string packet containing its `value` option value. This
+packet is sent to `IN` port of `console` node type that prints it to
+console. Option `prefix` of `console` node type is a prefix that is
+always added before sending text to console, so when running our server,
+now you'll see:
+
+```sh
+$ sol-fbp-runner http-light-server.fbp
+note: http-light-server running on port 8002, paths: /state and /power (string)
+```
+
+One thing that you may have noticed is that both nodes are named `_`.
+This means that they are anonymous nodes. Useful when nodes are used on
+a single connection on flow, so there's no need to name them.
+Finally, let's log values changes:
+
+```sh
+http_server OUT_POWER -> IN print_power(console)
+http_server OUT_STATE -> IN print_state(console)
+```
+
+## Tips about parsing errors
+
+If you write something that `sol-fbp-runner` fails to parse, don't panic!
+Small typos may always happen and Soletta tries to provide meaningful
+error messages to help us understand what is wrong and how to fix.
+Suppose we wrote something like:
+
+```sh
+power(http-server/imt:path="/power",port=8002)
+```
+
+You may get an error like:
+
+```sh
+$ sol-fbp-runner http-server.fbp
+WRN:sol-flow ./src/lib/flow/sol-flow-resolver-conffile.c:214 _resolver_conffile_get_module() Type='http-server/imt' not found.
+http-server.fbp:1:17 Couldn't resolve type 'http-server/imt' of node 'power'
+```
+
+Look at last line: it states the error - that it doesn't know about type
+`http-server/imt`. It also show the place of occurrence of first
+appearance of problematic node: `http-server.fbp:1:17`. We just have to
+find its declaration and fix the typo: node type is called
+`http-server/int`.
+Another common mistake is a typo at an option name, like:
+
+```sh
+_(constant/string:value="http-light-server running on port 8002, paths: /state and /power") OUT -> IN _(console:prefixo="note: ")
+```
+
+We then get an error like:
+
+```sh
+$ sol-fbp-runner http-server.fbp
+http-server.fbp:10:113 Unknown option name 'prefixo' of node '#anon:10:103'
+```
+
+As this error is on an anonymous node, it shows where it's declared:
+`'#anon:10:103'`.
+
+We can use this information to find node and make the proper fix: option
+name is `prefix`.
+
+This concludes our HTTP server. You should have something like this:
```

Comment:
create another heading/section here, right now looks like it is part of the error handling section.